internal/identifiers/identifiers.go

```diff
@@ -0,0 +1,29 @@
+package identifiers
+
+// Identifier represents a file identifier in an Archivematica
+// "identifiers.json" document.
+type Identifier struct {
+	Value string `json:"identifier"`
+	Type  string `json:"identifierType"`
+}
+
+// File represents a file and its associated identifiers in an Archivematica
+// "identifiers.json" document.
+type File struct {
+	Path        string       `json:"file"`
+	Identifiers []Identifier `json:"identifiers"`
+}
+
+// Compare returns an integer comparing two File paths lexicographically, for
+// sorting purposes. The result will be 0 if a.Path == b.Path, -1 if
+// a.Path < b.Path, and +1 if a.Path > b.Path.
+func Compare(a, b File) int {
+	switch {
+	case a.Path == b.Path:
+		return 0
+	case a.Path < b.Path:
+		return -1
+	default:
+		return 1
+	}
```

Comment:
I used `strings.Compare()` first, but the commentary contains this little disclaimer:
> // Use Compare when you need to perform a three-way comparison (with
> // [slices.SortFunc], for example). It is usually clearer and always faster
> // to use the built-in string comparison operators ==, <, >, and so on.

Which I took to mean "use comparison operators instead of strings.Compare()".  Re-reading that comment now though I think maybe the intended meaning is "use `strings.Compare()` only for three-way comparisons, but don't use it in situations where a single string operator would work instead.". :thinking:

Reply:
I read it like you are doing now, since it's only being used with `slices.SortFunc`.